scripts/git-utils/checkout-pr-if-exist.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
-# get all opened PRs  with specified label
 set -e
+# get all opened PRs with specified label
 RESULT=`curl -s https://api.github.com/search/issues?q=is:pr%20state:open%20label:$1%20repo:bio-tools/content`
 # check if PR is created by github-actions bot and return pull request-number
 PR_NUMBER=`jq '.items[]|select(.user.login == "github-actions[bot]")|.number' <<< $RESULT`
+# get the branch name and remove quotes
+TARGET_BRANCHNAME=`jq '.head.ref' <<< $(curl -s https://api.github.com/repos/OlegZharkov/content/pulls/$PR_NUMBER) | sed -r 's/^"|"$//g'`
```

Comment:
Is "OlegZharkov/content/" the right repo @OlegZharkov ?

Reply:
oh yeah, that was for debugging. Just a sec...

Reply:
should be better now @hmenager